ibis/tests/all/test_vectorized_udf.py

```diff
@@ -10,9 +10,22 @@ def add_one(s):
     return s + 1
 
 
+@elementwise(input_type=[dt.double], output_type=[dt.double])
+def add_one_with_output_type_list(s):
+    return s + 1
+
+
 @pytest.mark.only_on_backends([Pandas, PySpark])
 @pytest.mark.xfail_unsupported
 def test_elementwise_udf(backend, alltypes, df):
     result = add_one(alltypes['double_col']).execute()
     expected = add_one.func(df['double_col'])
     backend.assert_series_equal(result, expected, check_names=False)
+
+
+@pytest.mark.only_on_backends([Pandas, PySpark])
+@pytest.mark.xfail_unsupported
+def test_elementwise_udf_with_output_type_list(backend, alltypes, df):
```

Comment:
We should probably comment that this does NOT test the output of the UDF being and list. We only test that specifying output=[dt] is the same as output=dt in the udf decorator.